### PR TITLE
Updated setup instructions for Macbook M1 (with Mario GC)

### PR DESCRIPTION
**Pull request by fisher-alice**
## Summary

Working with Mario Gil Correa - we updated the set up instructions for Macbook M1 in SETUP.md - added another troubleshooting tip and pointed reader to Overview while following M1-specific instructions.

We included a link to the AWS Account Login - Getting AWS access for a new user google doc which has restricted access. We were unsure if this is okay or not. We could just point them to path of doc: Engineering > Wiki > Product.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -143,9 +148,9 @@ These steps are for OSX devices, including Apple Macbooks running on [Apple Sili
     1. Run `brew install openssl`
     2. Following the instructions in the output, run a form of `export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/`
 
-1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests.
-
-1. For Apple Silicon (M1), make these changes to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
+1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests. 
+  
+1. Return to the [Overview](#overview) to continue installation and clone the code-dot-org repo. Afer cloning, you will need to make the changes below to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
```
Comment (megcrenshaw) [inline SETUP.md]: So glad you added this part!

**Review by megcrenshaw — APPROVED** [review]
I know @cat5inthecradle did a bunch of work for documenting M1 stuff, and @Hamms recently made edits here as well. I'd check in with one of them. But it looks good to me! Thank you for the careful documentation here.

File: SETUP.md (hunk 1 above)
Comment (sanchitmalhotra126) [inline SETUP.md]: nit: *After

**Review by sanchitmalhotra126 — APPROVED** [review]
Thanks for updating this! +1 to Meg's comment about checking in with @cat5inthecradle and/or @Hamms but otherwise looks good to me!

**Comment by fisher-alice** [comment]
> I know @cat5inthecradle did a bunch of work for documenting M1 stuff, and @Hamms recently made edits here as well. I'd check in with one of them. But it looks good to me! Thank you for the careful documentation here.

Thanks - just added them as reviewers!

File: SETUP.md [hunk 2]
```diff
@@ -143,9 +148,9 @@ These steps are for OSX devices, including Apple Macbooks running on [Apple Sili
     1. Run `brew install openssl`
     2. Following the instructions in the output, run a form of `export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/`
 
-1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests.
-
-1. For Apple Silicon (M1), make these changes to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
+1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests. 
+
```
Comment (Hamms) [inline SETUP.md]: nit: unnecessary whitespace
```suggestion
1. Install [Google Chrome](https://www.google.com/chrome/), needed for some local app tests.

```

File: SETUP.md (hunk 1 above)
Comment (Hamms) [reply SETUP.md]: I believe this instruction is specific to M1 macs; I think it's worth keeping the `For Apple Silicon (M1),` bit

**Review by Hamms — APPROVED** [review]
A couple of minor nitpicks, but generally LGTM!
